src/main/java/com/dataloom/search/SearchApi.java

```diff
@@ -16,12 +16,26 @@
 	String ENTITY_TYPE_ID = "eid";
 	String PROPERTY_TYPE_ID = "pid";
 		
+	/*
+	 * The query, entityType, and propertyTypes params are all optional,
+	 * but at least one must be specified otherwise an error will be thrown.
+	 * All specified params are required to be present in each entity set returned.
+	 * If entityType and propertyTypes are both specified, the propertyTypes param
+	 * will be ignored.
+	 */
 	@GET( SEARCH )
 	String executeQueryJson(
```

Comment:
Please use javadoc format. If you press `Cmd + Alt + J` eclipse will auto generate the template for you.